FAQ.md

```diff
@@ -35,7 +37,9 @@
     *   `getting-started`: issue or pull request is suitable for someone new to GitHub and/or this project
     *   `help-wanted`: a question or request for assistance
     *   `left-as-was`: marks an issue closed because the item in question will be left as it was
```

Comment:
`left-as-was` like `duplicate` is better handled in the closing remarks of a pull request. Both are reasons for closing an issue or PR. I suggest removing.